assets/chat/js/chat.js

```diff
@@ -425,7 +424,11 @@ class Chat {
 
     withEmotes(emotes) {
         this.emoticons = new Set(emotes['destiny']);
-        this.twitchemotes = new Set(emotes['twitch']);
+        for (var s in generify_options) {
+            for (var e of this.emoticons) {
+                this.twitchemotes.add(`${e}:${s}`);    
+            }
+        }
```

Comment:
Why is this needed?

Reply:
It was part of the old solution. Now this.twitchemotes is not used anywhere

Reply:
It's confusing to me that we are using `this.twitchemotes` in a way that is completely unrelated to what the name implies. I think we should consider adding a new `Set` for this, or figure out if we even need a `Set` at all (is it for emote detection in places like [line 757](https://github.com/MemeLabs/chat-gui/blob/b27d45ae7ab496ccb62e0e5548e3d2d344dc9203/assets/chat/js/chat.js#L757)?).